gemini/function-calling/intro_function_calling.ipynb

```diff
@@ -56,6 +56,15 @@
     "</table>\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "| | |\n",
+    "|-|-|\n",
+    "|Author(s) | [Kris Overholt](https://github.com/koverholt) |"
```

Comment:
Could you change this from `Kris Overholt` to `Kristopher Overholt` in my notebooks. Much appreciated!